### ISSUE TITLE
cleos get info error

### ISSUE THREAD

**Issue by jrosich** [EOSIO/eos#3976]
Version v1.0.2 - node: eosbarcelona on Jungle tesnet

Installed new node. Account created. Connected to Jungle. Staked eos. Everything works but get account
```
./cleos.sh get account eosbarcelona
3345141ms thread-0   main.cpp:2659                 main                 ] Failed with error: unspecified (0)
unknown key
```
cleos.sh
```
CLEOS=/home/eos/eos-source/build/programs/cleos/cleos
$CLEOS -u http://127.0.0.1:8888 --wallet-url http://127.0.0.1:55553 "$@"
```

**Comment by vivekmarakana** [comment]
+1

**Comment by bt-cryptomancer** [comment]
confirmed this error is not specific to Jungle testnet.  Upgraded my node to https://github.com/EOS-Mainnet/eos/tree/launch-rc-1.0.2 on the 579a649aae8f660aa1abbab262437596d1f388f709b0b94a9fd6bba479889ea5 chain (BIOS Boot) and got same error:

./cleos.sh get account ge3tegenesis
54336ms thread-0   main.cpp:2659                 main                 ] Failed with error: Assert Exception (10)
itr != structs.end(): Unknown struct self_delegated_bandwidth

Interestingly no problem if you lookup the account by public key instead:

./cleos.sh get accounts EOS6WdawGE1Cega9sCW97WBjE4YgeggWWf5RsqrSVR5QY8A7cp8T3
{
  "account_names": [
    "ge3tegenesis"
  ]
}

Info about my node:

{"server_version":"1c560d2f","chain_id":"579a649aae8f660aa1abbab262437596d1f388f709b0b94a9fd6bba479889ea5","head_block_num":497000,"last_irreversible_block_num":496999,"last_irreversible_block_id":"00079567aa9a6f441e10f98ba6fb4c0facf08e087260e8d37818632d0462ae58","head_block_id":"00079568c1bc5890ef1e6c12258183eded3bef4d59d0dbc6158b5fd929d3a8fc","head_block_time":"2018-06-07T18:15:05","head_block_producer":"eosio","virtual_block_cpu_limit":200000000,"virtual_block_net_limit":1048576000,"block_cpu_limit":199900,"block_net_limit":1048576}

**Comment by Heejaejoo** [comment]
From what I investigated from eosio code, I found this is a bug related to the recent change.
According to line 1003~1042 of chain_plugin.cpp, cleos initializes abi_serializer( abi_serializer abis( abi ))
) with a contract from eosio account, which definitely should be the eosio.system contract.
Then, cleos loads self_delegated_bandwidth struct from the abi(eosio.system) in line 242 of abi_serializer.cpp, which has no struct named self_delegated_bandwidth and causes the error.

**Comment by JohnnyZhao** [comment]
same here, is there fix on the way?

**Comment by livc** [comment]
same here...

**Comment by zhaoxingjie** [comment]
same here...

**Comment by wanderingbort** [comment]
this seems to be resolved by 843ed6452f6bfc7deb74b660dfcd7758ba6a1775
